core/runtime.rs

```diff
@@ -1110,7 +1097,8 @@ impl JsRuntime {
           v8::PromiseState::Fulfilled => {
             state.pending_mod_evaluate.take();
             scope.perform_microtask_checkpoint();
-            sender.try_send(Ok(())).unwrap();
+            // Receiver end might have been already dropped, ignore the result
+            let _ = sender.try_send(Ok(()));
```

Comment:
Can the unwrap panic being fixed here be demonstrated with a test in deno_core?

Reply:
It is - tests in `deno_core` don't really await on the receiver from `mod_evaluate`

Reply:
I'm suggesting to add a test that would panic on master branch, but is fixed here by removing the unwrap.

Reply:
@ry that's not really possible because the need to remove unwrap arises from this PR - previously `mod_evaluate` returned a future that held a reference to the receiver until it was resolved, in this PR receiver handle is returned to the caller which might drop it in which case `unwrap()` would panic. This case is covered in test in `core/runtime.rs` and `core/modules.rs` which all drop receiver handle and there's no panic.

Reply:
Ok fair